pubnub-kotlin/pubnub-kotlin-test/src/commonTest/kotlin/com/pubnub/api/UserMetadataTest.kt

```diff
@@ -141,4 +146,85 @@ class UserMetadataTest : BaseIntegrationTest() {
             assertEquals(uuid, message.uuid)
         }
     }
+
+    @Test
+    fun can_get_metadata() = runTest(timeout = 10.seconds) {
+        // given
+        pubnub.setUUIDMetadata(
+            uuid,
+            name = name,
+            externalId = externalId,
+            profileUrl = profileUrl,
+            email = email,
+            status = status,
+            custom = custom,
+            includeCustom = includeCustom,
+            type = type
+        ).await()
+
+        //when
+        val result = pubnub.getUUIDMetadata(uuid, includeCustom = true).await()
+
+        // then
+        val pnuuidMetadata = result.data
+        requireNotNull(pnuuidMetadata)
+        assertEquals(uuid, pnuuidMetadata.id)
+        assertEquals(name, pnuuidMetadata.name)
+        assertEquals(externalId, pnuuidMetadata.externalId)
+        assertEquals(profileUrl, pnuuidMetadata.profileUrl)
+        assertEquals(email, pnuuidMetadata.email)
+        assertEquals(status, pnuuidMetadata.status)
+        assertEquals(customData, pnuuidMetadata.custom)
```

Comment:
FYI, there is no assertion for `includeCustom`

Reply:
includeCustom is not returned in the response, just `custom` or empty